R/input-date.R

```diff
@@ -94,8 +100,7 @@
 dateInput <- function(inputId, label, value = NULL, min = NULL, max = NULL,
   format = "yyyy-mm-dd", startview = "month", weekstart = 0,
   language = "en", width = NULL, autoclose = TRUE,
-  datesdisabled = NULL, daysofweekdisabled = NULL,
-  sassVariables = list()) {
+  datesdisabled = NULL, daysofweekdisabled = NULL, ..., sassVars = list()) {
```

Comment:
Should also call `ellipsis::check_dots_empty()` in this function to make sure there are no `...` args.

Reply:
Do we already depend on that package? If not, we could also implement our own?

Reply:
We don't import it yet, but the package is lightweight and maintained by Hadley and Lionel, and it's used in tidyverse packages, so I think it's OK to bring it in.

Reply:
No longer relevant now that we're no longer doing `sassVars`